apps/emqx/test/emqx_common_test_helpers.erl

```diff
@@ -1260,7 +1260,12 @@ call_janitor() ->
 
 call_janitor(Timeout) ->
     Janitor = get_or_spawn_janitor(),
-    ok = emqx_test_janitor:stop(Janitor, Timeout),
+    try
+        emqx_test_janitor:stop(Janitor, Timeout)
+    catch
+        K:E:S ->
+            ct:pal("janitor crashed:\n  ~p", [{K, E, S}])
+    end,
```

Comment:
I feel there's a risk of hiding issues introduced by testcase authors that might pop up later in unrelated testcases or even test suites. After all, `end_per_testcase is crashed` is more visible that a log entry hiding inside green testcase output.

Reply:
Fair.  I reverted this and only ignored 5xx errors in the s3tables suite to have the desired effect.